services/api-server/tests/unit/api_solvers/test_api_routers_solvers_jobs_delete.py

```diff
@@ -174,9 +183,20 @@ async def test_create_job(
     callback = mock_webserver_router["create_projects"].side_effect
 
     def create_project_side_effect(request: httpx.Request):
+        # check `hidden` bool
         query = dict(elm.split("=") for elm in request.url.query.decode().split("&"))
         _hidden = query.get("hidden")
         assert _hidden == ("true" if hidden else "false")
+
+        # check parent project and node id
+        if parent_project_id is not None:
+            assert f"{parent_project_id}" == dict(request.headers).get(
+                "x-simcore-parent-project-uuid"
```

Comment:
use the constant you define here as well.

Reply:
done